dips/models.py

```diff
@@ -263,7 +265,9 @@ def requires_es_descendants_delete(self):
 
 
 class DIP(AbstractEsModel):
-    objectszip = models.FileField(_("objects zip file"))
+    objectszip = models.FileField(
+        _("DIP file"), help_text=_("Related DIP (only tar and zip files allowed).")
+    )
```

Comment:
I was going to add Django's `FileExtensionValidator` to this filed, but since that's not more than a check and the DIP forms are only accessible for administrators and editors, I thought it was unnecessary. The `extract_mets` task will fail if the file can't be handled by `tarfile` or `zipfile`.